discord/ext/test/__init__.py

```diff
@@ -7,4 +7,5 @@
 
 from . import backend
 from .runner import *
+from .utils import *
```

Comment:
Maybe instead explicitly import only the methods, to reduce the chance of more accidental imports

Reply:
idk which ones people use though, I can go through and import all the methods, or are there some you want not imported?

Reply:
I was thinking just explicitly import all the methods